opensearch-mixin/config.libsonnet

```diff
@@ -1,8 +1,9 @@
 {
   _config+:: {
+    enableMultiCluster: false,
     // extra static selector to apply to all templated variables and alerts
-    filteringSelector: 'cluster!=""',
-    groupLabels: ['job', 'cluster'],
+    filteringSelector: if self.enableMultiCluster then 'cluster!="",opensearch_cluster!=""' else 'opensearch_cluster!=""',
```

Comment:
Sorry not to familiar with recent k8 stuff, do we usually change the filtering selector with these k8 updates?

Reply:
@BominRahmani In this case, the metrics are exported with a `cluster` label by default and we've opted to replace that label with `opensearch_cluster`.

The reason for this is that the `cluster` label, in the context of k8s, is meant to represent the k8s cluster. Keeping the label as `cluster` when the deployment is not k8s based seemed inconsistent.